app/views/devise/registrations/edit.html.erb

```diff
@@ -4,6 +4,11 @@
   <%= devise_error_messages! %>
 
   <div class="field">
+    <%= f.label :avatar %><br />
+    <%= f.file_field :avatar, autofocus: true %>
```

Comment:
UI wise, we typically see this at the bottom of a form. You should only have one `autofocus` input per form.